packages/docs/src/routes/qwikcity/routing/index.mdx

```diff
@@ -42,8 +42,8 @@ src/
 
 - **`[id]`** is a directory that represents a dynamic route segment, in this example `id` is the param.
 - **`[...catchall]`** is a directory that represents a dynamic catch-all route, in this example `catchall` is the param.
-- **`index.ext` files** are the pages/endpoints.
-- **`layout.ext` files** are the layouts.
+- **`index.tsx` files** are the pages/endpoints.
```

Comment:
I don't think it's a typo, because the `index` files can have either `.tsx` or `.mdx` extensions (as seen by the changed file name itself)

mind changing this to include this explanation? (that `ext` can be either `tsx` or `mdx`)

thanks!